.gitignore

```diff
@@ -1,9 +1,10 @@
-node_modules
-npm-debug.log
-.DS_Store
 *.sw[po]
 *.xpi
-logs/
-dist/
+.DS_Store
+config.yml
```

Comment:
Added this "config.yml" bit. The rest of the noise was A-Z sorting for "easier" readability.